assignment3.txt

```diff
@@ -0,0 +1,6 @@
+Lloyd Page
+1. Yes, however i must be declare and initialized before the for loop
```

Comment:
You're right about that, but take a look at that print statement again - it's not quite right!